### PR TITLE
Editing revisions for typos, grammar, clarity

### PR DESCRIPTION
**Pull request by chungkky**
1. Made proof reading fixes.

### CODE REVIEW

File: 00-intro.md [hunk 1]
```diff
@@ -94,12 +94,12 @@ She has 300 samples in all, and now needs to:
...
     with corresponding statistics for each other protein
     using a program one of the other graduate students wrote called `goodiff`.
-4.  Write up.
+4.  Write up results.
     Her supervisor would really like her to do this by the end of the month
     so that her paper can appear in an upcoming special issue of *Aquatic Goo Letters*.
 
 It takes about half an hour for the assay machine to process each sample.
-The good news is,
+The good news is:
```
Comment (gvwilson) [inline 00-intro.md]: Hm - I'd use a comma here.

File: index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ These lessons will start you on a path towards using these resources effectively
 
 > ## Prerequisites {.prereq}
 >
-> This lesson guides you through the basics of filesystems and the
+> This lesson guides you through the basics of file systems and the
```
Comment (gvwilson) [inline index.md]: You're right, we're inconsistent - 17 uses of 'filesystem' (one word) in our lessons, 12 uses of 'file system' (two words).  In the shell novice lesson, though, it's 11 to 12, so yes, make the change.

File: 00-intro.md (hunk 1 above)
Comment (ChristinaLK) [inline 00-intro.md]: I wouldn't have any punctuation at all!  ;)

**Comment by ChristinaLK** [comment]
looks good - will merge later today

**Pull request merged**